### PR TITLE
CABAM implementation for mwe node classification

### PR DESCRIPTION
**Pull request by muzz-yasir**
Cleaned up commit history from [previous](https://github.com/google-research/graphworld/pull/9) (closed) PR into new branch for merging.

### CODE REVIEW

File: src/generate_stochastic_block_model_with_features.py [hunk 1]
```diff
@@ -25,7 +25,11 @@
 
 import numpy as np
 
+<<<<<<< HEAD
```
Comment (jpalowitch) [inline src/generate_stochastic_block_model_with_features.py:28]: This merge boilerplate guidance needs removing

Comment (jpalowitch) [inline src/generate_stochastic_block_model_with_features.py:28]: This merge boilerplate should be removed. Also, both the mwe and local test runs should have failed if these lines are here. Did you test either the mwe or local test runs after this last commit?

File: src/graph_world/generators/cabam_simulator.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      # Look up the vertex structs from our vertices mapping and add edge.
+      e = cabam_data.graph.add_edge(vertices[src], vertices[dst])
+
+  
+  return cabam_data
+
+
+def GenerateAssortativityDict(p_in, assortativity_type, temperature):
+    """
+    Generates a dictionary representing the Assortativiity Constant in CABAM generation - the parameter named 'c_probs'.
```
Comment (jpalowitch) [inline src/graph_world/generators/cabam_simulator.py:78]: "Assortativity" has an extra 'i'

**Review by jpalowitch — CHANGES_REQUESTED** [review]
Thanks Mustafa, LGTM except for a hiccup with some merge boilerplate. Can you remove that and check that the MWE works (for both CABAM and SBM)? Thanks!

File: src/generate_stochastic_block_model_with_features.py (hunk 1 above)
Comment (muzz-yasir) [reply src/generate_stochastic_block_model_with_features.py]: @jpalowitch Apologies for missing out on that! Afaik the main_local and main_local_mwe runs don't call this file at any point and it is just a standalone test script? 
Mentioning as both runs worked fine when testing that commit and I can't find an entrypoint into the file from the main runs.

**Pull request merged**